content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -52,22 +52,20 @@ route, we recommend you add IP route(s) so Kubernetes cluster addresses go via t
 
 ### Master node(s)
 
-| Protocol | Direction | Port Range | Purpose                 |
-|----------|-----------|------------|-------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   |
-| TCP      | Inbound   | 2379-2380  | etcd server client API  |
-| TCP      | Inbound   | 10250      | Kubelet API             |
-| TCP      | Inbound   | 10251      | kube-scheduler          |
-| TCP      | Inbound   | 10252      | kube-controller-manager |
-| TCP      | Inbound   | 10255      | Read-only Kubelet API   |
+| Protocol | Direction | Port Range | Purpose                 | Source Components         |
```

Comment:
does "source components" here mean "components that need access?" I think that could be a little clearer. "Clients" maybe?

Reply:
@liztio "Clients" is better.
@luxas @timothysc Any suggestions? 
Naming is always the hardest part.

Reply:
"used by" ?